gensrc/thrift/FrontendService.thrift

```diff
@@ -1185,6 +1185,7 @@ enum TBinlogType {
   TRUNCATE_TABLE = 13,
   RENAME_TABLE = 14,
   RENAME_COLUMN = 15,
+  MODIFY_VIEW_DEF = 16,
```

Comment:
16 -> 17 (conflicts with master branch)

Reply:
Thanks for your suggestion, I've dealt with it